volplugin/handlers.go

```diff
@@ -168,6 +173,9 @@ func mount(master, host string) func(http.ResponseWriter, *http.Request) {
 			return
 		}
 
+		stopChan := addStopChan(uc.Request.Name)
+		go heartbeatMount(master, ttl, ut, stopChan)
```

Comment:
interesting. I didn't think about this, that's a bit harder problem. 

At initial thought I wondered if we could depend on ceph to fail a duplicate mount call. But I think that won't work if the both the competing mount calls are made from same host?

We can leave this as is until we find a good solution, but might be good to add a comment just to remind us in future :)